### PR TITLE
Add logging for ironic-inspector to shared directory

### PR DESCRIPTION
**Pull request by bfournie**
Set the inspector logging to /shared. Also enable logging on the ramdisk running introspection.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,15 +9,16 @@ RUN mkdir -p /var/lib/ironic-inspector && \
 
 RUN crudini --set /etc/ironic-inspector/inspector.conf DEFAULT auth_strategy noauth && \ 
     crudini --set /etc/ironic-inspector/inspector.conf ironic auth_strategy noauth && \
-    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug true && \
+    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug false && \
```
Comment (elfosardo) [inline Dockerfile]: just wondering if we should leave the debug level anyway

**Review by elfosardo — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (dtantsur) [reply Dockerfile]: Let's keep debug logging at this stage of development

File: Dockerfile [hunk 2]
```diff
@@ -9,15 +9,16 @@ RUN mkdir -p /var/lib/ironic-inspector && \
...
     crudini --set /etc/ironic-inspector/inspector.conf database connection sqlite:///var/lib/ironic-inspector/ironic-inspector.db && \
     crudini --set /etc/ironic-inspector/inspector.conf DEFAULT transport_url fake:// && \
     crudini --set /etc/ironic-inspector/inspector.conf processing store_data database && \
+    crudini --set /etc/ironic-inspector/inspector.conf processing ramdisk_logs_dir /shared/log/ironic-inspector/ramdisk && \
+    crudini --set /etc/ironic-inspector/inspector.conf processing always_store_ramdisk_logs true && \
```
Comment (dtantsur) [inline Dockerfile:17]: In case we haven't done it already, we should pass `ipa-debug=1` to the ramdisk.

File: Dockerfile (hunk 1 above)
Comment (bfournie) [reply Dockerfile]: Changed debug back to true.

File: Dockerfile (hunk 2 above)
Comment (bfournie) [reply Dockerfile:17]: Yes, we are setting that in inspector.ipxe - https://github.com/metalkube/metalkube-ironic/blob/master/inspector.ipxe#L7

**Review by dtantsur — APPROVED** [review]

**Pull request merged**